index.bs

```diff
@@ -1657,16 +1684,20 @@ To <dfn>generate an attribution debug report URL</dfn> given an [=attribution de
 
 <h3 id="create-report-request">Creating a report request</h3>
 
-To <dfn>create a report request</dfn> given a [=URL=] |url| and a [=byte sequence=] |body|:
+To <dfn>create a report request</dfn> given a [=URL=] |url|, a [=byte sequence=] |body|,
+and a [=header list=] |newHeaders|:
```

Comment:
We should default `newHeaders` to the empty list, or else some other invocations of itthis algorithm in the spec don't "type check".

Reply:
Done